client/src/lib/components/DatasetListItem/DatasetListItem.svelte

```diff
@@ -16,104 +16,118 @@
   };
 </script>
 
-<li class="fr-p-3w">
-  <p class="fr-logo fr-logo--sm fr-p-0" title="république française">
-    {@html "Ministère<br />de la culture"}
-  </p>
-
-  <div class="fr-container">
-    <p class="fr-m-0">
-      {#if dataset.headlines}
-        <strong data-testid="headlines-title">
-          {@html dataset.headlines.title}
-        </strong>
-      {:else}
-        <strong>
-          {dataset.title}
-        </strong>
-      {/if}
+<li class="fr-p-3w container">
+  <a
+    data-test-id={`${dataset.title}`}
+    href={paths.datasetDetail({ id: dataset.id })}
+    title="Consulter cette fiche de données"
+  >
+    <p class="fr-logo fr-logo--sm fr-p-0" title="république française">
+      {@html "Ministère<br />de la culture"}
     </p>
 
-    <p class="fr-m-0 fr-text--sm fr-text-mention--grey">
-      {dataset.service}
-    </p>
+    <div class="container">
```

Comment:
Je ne comprends pas pourquoi avoir utilisé une classe custom équivalant à `width: 100%` ? Si je remets ici `fr-container` en retirant `container` dans le `li`, la mise en page du list item prend bien tout l'espace de la page. Et ce avec une bordure entre le logo du ministère et les infos qui correspond mieux à la maquette.

Par ailleurs, la largeur de la colonne contenant le logo est désormais insuffisante, car on a maintenant 3 lignes : "Ministère / de la / culture". Le layout grid précédent avec une colonne de taille `auto` assurait un espace suffisant pour respecter le `@html` ci-dessus.

Reply:
alors en fait la classe `fr-container` utilise tout un tas de classe css (dont tu padding, margin, ...) dont j'ai pas besoin. Ici cette classe `.container` n'a qu'une prop css : flex-grow.

Au besoin (si tu ne connais pas déja) tu peux check https://css-tricks.com/snippets/css/a-guide-to-flexbox/ c'est un peu ma bible